docs/release/release-tasks.md

```diff
@@ -289,6 +289,8 @@ The goal of this task to make the book for the current release available under e
     6. Sort entries within a section alphabetically. 
     7. Write highlights section based on the initial release notes doc.
     8. Add Kubernetes version support section.
+    9. Modify `Changes since v1.x.y` to `Changes since v1.x`
```

Comment:
Just adding a note seems okay for now. Modifying the release notes tool for this specific use case (.0) release isn't worth the effor imho